fe1-web/jest/setup.js

```diff
@@ -0,0 +1,17 @@
+import 'react-native-gesture-handler/jestSetup';
+import { jest } from '@jest/globals';
+
+jest.mock('react-native-reanimated', () => {
+  // eslint-disable-next-line global-require
+  const Reanimated = require('react-native-reanimated/mock');
+
+  // The mock for `call` immediately calls the callback which is incorrect
+  // So we override it with a no-op
+  Reanimated.default.call = () => {};
```

Comment:
Sonar is complaining about this.
I'd suggest fixing it in the simplest way, by adding a comment in the arrow function to clarify the intent.

```suggestion
  Reanimated.default.call = () => { /* no-op */ };
```